project-base/storefront/components/Pages/BlogCategory/BlogCategoryArticlesWrapper.tsx

```diff
@@ -39,9 +41,11 @@ export const BlogCategoryArticlesWrapper: FC<BlogCategoryArticlesWrapperProps> =
                 <BlogArticlesList blogArticles={mappedArticles} />
             ) : (
                 <div className="flex flex-col gap-10">
-                    {createEmptyArray(DEFAULT_PAGE_SIZE).map((_, index) => (
-                        <SkeletonModuleArticleBlog key={index} />
-                    ))}
+                    {areBlogCategoryArticlesFetching ? (
```

Comment:
Can this condition be somehow rewritten so that `areBlogCategoryArticlesFetching` is not used twice (lines 40 and 44)?

Reply:
that is a great idea! 6b66e74125 (thank you chatgpt :smile: )